copy-this-template-directory/index.md

```diff
@@ -1,16 +1,1050 @@
 ---
-title: ''
-description: '120-160 characters'
+title: 'Auth with Directus and iOS'
+description: 'Directus auth for an iOS project'
 author:
-  name: ''
-  avatar_file_name: 'add-to-directory'
+  name: 'Harshpal Bhirth'
+  avatar_file_name: 'Harshpal.jpeg'
 ---
 
 ## Introduction
 
+In this tutorial, you will learn how to configure an iOS project that authenticates with Directus. You'll cover registering, logging in, logging out, viewing all posts from all users, creating a post, and editing and deleting posts from your user account.
+
 ## Before You Start
 
-<!-- ## Your Sections Here -->
+You will need:
+
+1. To have Xcode installed on your macOS machine.
+2. Knowledge of the Swift programming language.
+3. A Directus project - follow our [quickstart guide](docs.directus.io/getting-started/quickstart) if you don't already have one.
+
+## Setting Up Public Role
+
+1. Navigate to `Settings`.
+2. Select `Access Control`.
+3. Click on the `Public Role`.
+4. Access the `System Collection`.
+5. Locate `Directus_users`.
+6. Under the `Create` option, choose `Use Custom`.
+7. Enable field permissions for `Email` and `Password` options.
+
+## Creating A New Role 
+
+1. On the Access Control page, click the plus button.
+2. Name your new role, such as `iOS App User`.
+3. Under the "Post" collection, enable `Create` and `Read`.
+4. For `Write` and `DELETE`, select `Use Custom`.
+5. Add Filter `user_created -> id Equals $CURRENT_USER.id` for both update and delete options.
+6. This configuration ensures users can create and read all posts, but only update and delete their own posts.
+7. Copy the Primary Key from the iOS App User role.
+8. In the Public Role settings, under `Directus_users`, select `Use Custom` for the create option.
+9. Click `Field Presets` and paste the Primary Key.
+
+By following these steps, you'll set up a role in Directus that grants users of your iOS app the ability to view all posts but restricts editing and deleting permissions to posts associated with their authenticated user account.
+
+
+## Content View
```

Comment:
You're missing a bit of text under this that explains what we'll be doing / what to do with this code

Reply:
Lets also add a screenshot here to make it easier to follow.